ow_plexil/src/plexil-adapter/PlexilInterface.cpp

```diff
@@ -10,11 +10,23 @@ using std::string;
 using std::map;
 using std::make_unique;
 
-// Dummy operation ID that signifies idle lander operation.
-#define IDLE_ID (-1)
+///////////////////////// Utilities /////////////////////////
+
+static double normalize_degrees (double angle)
```

Comment:
My range comment was intended for this function. Do you know what ranges of angles this results in? Can you put it in a comment?

Reply:
I'd have to study it, and this function has been around long before this PR.  I've made a to-do to add the comment for Release 12 (comments don't need PRs) but suggest leaving it out of this review.